test/expected/parallel-10.out

```diff
@@ -98,79 +106,163 @@ LIMIT 5;
 (1 row)
 
-EXPLAIN (costs off) SELECT histogram(i, 0,100000,5) FROM "test";
-                          QUERY PLAN                           
----------------------------------------------------------------
- Finalize Aggregate
-   ->  Gather
+:PREFIX SELECT histogram(i, 0,100000,5) FROM "test";
+                                         QUERY PLAN                                         
+--------------------------------------------------------------------------------------------
+ Finalize Aggregate (actual rows=1 loops=1)
+   ->  Gather (actual rows=3 loops=1)
          Workers Planned: 2
-         ->  Partial Aggregate
-               ->  Append
-                     ->  Parallel Seq Scan on test
-                     ->  Parallel Seq Scan on _hyper_1_1_chunk
-                     ->  Parallel Seq Scan on _hyper_1_2_chunk
-(8 rows)
+         Workers Launched: 2
+         ->  Partial Aggregate (actual rows=1 loops=3)
+               ->  Append (actual rows=333333 loops=3)
+                     ->  Parallel Seq Scan on test (actual rows=0 loops=3)
+                     ->  Parallel Seq Scan on _hyper_1_1_chunk (actual rows=166667 loops=3)
+                     ->  Parallel Seq Scan on _hyper_1_2_chunk (actual rows=166667 loops=3)
+(9 rows)
 
 SELECT histogram(i, 0, 100000, 5) FROM "test";
                 histogram                 
 ------------------------------------------
  {0,20000,20000,20000,20000,20000,900000}
 (1 row)
 
-EXPLAIN (costs off) SELECT histogram(i, 10,100000,5) FROM "test";
-                          QUERY PLAN                           
----------------------------------------------------------------
- Finalize Aggregate
-   ->  Gather
+:PREFIX SELECT histogram(i, 10,100000,5) FROM "test";
+                                         QUERY PLAN                                         
+--------------------------------------------------------------------------------------------
+ Finalize Aggregate (actual rows=1 loops=1)
+   ->  Gather (actual rows=3 loops=1)
          Workers Planned: 2
-         ->  Partial Aggregate
-               ->  Append
-                     ->  Parallel Seq Scan on test
-                     ->  Parallel Seq Scan on _hyper_1_1_chunk
-                     ->  Parallel Seq Scan on _hyper_1_2_chunk
-(8 rows)
+         Workers Launched: 2
+         ->  Partial Aggregate (actual rows=1 loops=3)
+               ->  Append (actual rows=333333 loops=3)
+                     ->  Parallel Seq Scan on test (actual rows=0 loops=3)
+                     ->  Parallel Seq Scan on _hyper_1_1_chunk (actual rows=166667 loops=3)
+                     ->  Parallel Seq Scan on _hyper_1_2_chunk (actual rows=166667 loops=3)
+(9 rows)
 
 SELECT histogram(i, 10, 100000, 5) FROM "test";
                  histogram                 
 -------------------------------------------
  {10,19998,19998,19998,19998,19998,900000}
 (1 row)
 
+-- test constraint aware append
```

Comment:
should we test that now() does not lead to a parallel plan? Just for completeness?

Reply:
added